app/views/clients/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
```

Comment:
what is the difference between container and container fluid and if we are using this as a start in each view, should it now be in layout?